reduce and reuse maz to maz network impedance calls to speed up runtime

bstabler opened the pull request:
See #431.  We'll want to pull in these example expression file improvements once ready.

bstabler committed:
reduce and reuse maz to maz network impedance calls to speed up runtime

esanchez01 commented:
@bstabler, @wusun2 and I are interested in knowing whether this change improves the 1 and 3 zone runtimes as well. I tested the 1 zone system on our machine but the runtimes varied quite heavily from run to run. From our past discussions/shared runtimes, your machine is more consistent -- have you tested the 1 and 3 zone systems on your machine?

esanchez01 committed:
Merge branch 'ActivitySim:fewer-maz-maz-data-calls-for-sandag-two-zone' into fewer-maz-maz-data-calls-for-sandag-two-zone

esanchez01 committed:
Apply Ben's maz-maz update to 1 and 3 zone setups

esanchez01 commented:
@bstabler, per our discussion, I applied these changes to the 1 and 3 zone setups as well. You can find the updated setups here for testing: https://github.com/esanchez01/activitysim/tree/fewer-maz-maz-data-calls-for-sandag-two-zone

bstabler committed:
Merge remote-tracking branch 'remotes/origin/develop' into fewer-maz-maz-data-calls-for-sandag-two-zone

Commit:
Merge pull request #510 from esanchez01/fewer-maz-maz-data-calls-for-sandag-two-zone

Fewer maz maz data calls for sandag two zone

bstabler commented:
thanks @esanchez01, I merged your revisions

bstabler committed:
small updates to the expected test results

bstabler committed:
updated test results

bstabler committed:
merge remote-tracking branch 'remotes/origin/develop'

update test results

save outputs to make tests with multiple tests easier to update

bstabler committed:
pycodestyle
